hmy/staking.go

```diff
@@ -519,7 +519,7 @@ func (hmy *Harmony) GetDelegationsByDelegatorByBlock(
 }
 
 // UndelegationPayouts ..
-type UndelegationPayouts map[common.Address]*big.Int
+type UndelegationPayouts map[common.Address]map[common.Address]*big.Int
```

Comment:
Is this data exposed to other RPC call, if you changed it here, will it break other API, like staking dashboard?  @potvik

Reply:
@JackyWYX has a different view on this data structure to improve readability and maintainability.  He suggests changing to struct, instead of a map of map.

Reply:
Thanks @JackyWYX for the suggestion, I convert it to a data struct and make a function to update its internal data. Hope this can help future extension or modification.

Reply:
Surely it will. Thanks!